src/config/config.ts

```diff
@@ -0,0 +1,3 @@
+export default () => ({
+    db: process.env.DB ?? "mongodb://localhost/nest"
+  });
```

Comment:
The file is not necessary. Because i configed it in [this file](https://github.com/meeorder/backend/blob/develop/src/config/index.ts)

Reply:
Okay, got it

Reply:
deleted